src/Components/Components.sln

```diff
@@ -263,6 +263,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Wasm.Authentication.Shared"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.JSInterop", "..\JSInterop\Microsoft.JSInterop\src\Microsoft.JSInterop.csproj", "{A062ACCE-AB0D-4569-9548-4BC0D9A9174B}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Web.Extensions", "Web.Extensions", "{EDA58361-CABB-4050-9F5E-5889A260DB61}"
```

Comment:
Do we need both Microsoft.AspNetCore.Components.Web.Extensions and Web.Extensions or was this left over from an earlier version?

Reply:
Think that's the folder